network-api/networkapi/tests/test_users.py

```diff
@@ -0,0 +1,60 @@
+from django.contrib.auth.models import Group, User
+from django.core.management import call_command
+from django.test import TestCase
+
+
```

Comment:
Considering that these tests are for a management command that is defined in the `network-api/networkapi/management` app, have you considered adding moving this test module into that app as well (e.g. `network-api/networkapi/management/tests/test_delete_non_staff.py`)?

Reply:
Hi @tbrlpld this move makes sense to me also **kudos** I think that the rename is a great idea considering that all of these tests are testing the `delete_non_staff` command!